test/JDBC/input/BABEL_4012.sql

```diff
@@ -0,0 +1,233 @@
+CREATE SCHEMA BABEL_4012_sch
+GO
+
+CREATE TABLE BABEL_4012_sch.table1(a int, b int);
+GO
+
+INSERT INTO BABEL_4012_sch.table1 VALUES(1,4), (1,5);
+GO
+
+-- WITH ALIAS
+
+-- with schema name in set and update
+UPDATE BABEL_4012_sch.table1 SET BABEL_4012_sch.table1.a = 2 FROM BABEL_4012_sch.table1 AS t1
```

Comment:
Add few test cases with alias in quoted identifier and have a mixed cases in the alias along with multibyte chars